citest/json_contract/value_observation_verifier.py

```diff
@@ -80,22 +80,25 @@ def add_constraint(self, constraint):
 
   def contains_path_match(self, path, match_spec, min=1, max=None,
                           **kwargs):
+    match_kwargs = kwargs.pop('match_kwargs', {})
     if isinstance(match_spec, list):
```

Comment:
If `match_spec` is neither a list or a dict, this will fail inside the dict branch with some nonsensical exception. Can you make the failure where the `match_spec` is neither a list or dict more explicit?